app/common/configs/cache-config.js

```diff
@@ -2,7 +2,7 @@ module.exports = ['CacheFactoryProvider', function (CacheFactoryProvider) {
     angular.extend(CacheFactoryProvider.defaults, {
         maxAge: 15 * 60 * 1000, // 15 mins
         cacheFlushInterval: 60 * 60 * 1000, // This cache will clear itself every hour.
-        storageMode: 'localStorage',
+        storageMode: 'sessionStorage',
         storagePrefix: 'ush-caches.',
```

Comment:
btw, I'm just realising this prefix and storage approach probably dooms the installation of two deployments under the same domain? i.e. mydomain.com/deployment1 , mydomain.com/deployment2

not a critique or something we have to support, just a realisation moment